onelogin_rules/onelogin_user_account_suspended.py

```diff
@@ -0,0 +1,14 @@
+def rule(event):
+
+    # check that this is a user suspended event
+    # event 21 is user suspended event; event 551 is user suspended via api
```

Comment:
I might actually combine "User locked by API" and "User suspended by API" into one rule, they seem to be looking at the same type of behavior/activity.